web_ui/src/components/ActivityPage.tsx

```diff
@@ -63,16 +63,12 @@ function ActivityPageInner({ data }: IActivityPageInnerProps) {
 
 interface IActivityPageContainer {
   readonly children: React.ReactNode
-  readonly className?: string
 }
-function ActivityPageContainer({
-  children,
-  className,
-}: IActivityPageContainer) {
+function ActivityPageContainer({ children }: IActivityPageContainer) {
   return (
-    <Container className={className}>
+    <div className="flex-grow-1 d-flex flex-column">
```

Comment:
the container here was adding extra padding as well, not necessary since the page is already wrapped in a `container`